kpq/keyed_priority_queue.go

```diff
@@ -104,6 +113,12 @@ func (pq *KeyedPriorityQueue[K, V]) Push(k K, v V) error {
 }
 
 func (pq *KeyedPriorityQueue[K, V]) push(k K, v V) {
+	select {
+	case pq.fastTrack <- pair[K, V]{k: k, v: v}:
```

Comment:
As I got it, fast track omits calculating priority for an item or adding it to a queue. 
What do we want to use it for?

Reply:
If there was a call by reader to `BlockingPop()` first, we will use `fastTrack` to transfer the element to the reader directly, so reader can be unblocked and can return the result.